codegen/luajit/runtime/runtime.lua

```diff
@@ -754,6 +754,19 @@ do
 		ffi.copy(start, data, len or #data)
 	end
 
+	function store.copy(memory, dst, src, len)
```

Comment:
The `load` and `store` groupings generally deal in types so `copy` and `fill` don't quite mesh well. I would say to put it under `allocator` because it deals with the memories themselves, but I might need to also rename that to better express it.